### PR TITLE
Update README.md about changing project name

### PR DESCRIPTION
**Pull request by nerrons**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,7 +32,10 @@ This template is the result of learnings from many previous projects and should
 
 ### Adjust the template to your needs
 
-- Use this repo [as a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template) and replace all occurrences of "Greeter" in the relevant CMakeLists.txt with the name of your project
+- Use this repo [as a template](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-from-a-template).
+- Replace all occurrences of "Greeter" in the relevant CMakeLists.txt with the name of your project
+  - Note that capitalization matters here: "Greeter" means the name of the project, while "greeter" is used in file names!
```
Comment (TheLartians) [inline README.md:37]: As we're talking about code changes I think code markdown may be more appropriate.

```suggestion
  - Capitalization matters here: `Greeter` means the name of the project, while `greeter` is used in file names.
```

File: README.md [hunk 2]
```diff
@@ -32,7 +32,10 @@ This template is the result of learnings from many previous projects and should
...
+  - Change `#include <greeter/version.h>` to `#include <YourNewProjName/version.h>` in all .cpp files, if you're using them
```
Comment (TheLartians) [inline README.md:38]: How about we formulate it more generally, something like the following?

```suggestion
  - Remember to rename the `include/greeter` directory to use your project's lowercase name and update all relevant `#include`s accordingly.
```

**Review by TheLartians — COMMENTED** [review]
Awesome, thanks for initiating the PR!

File: README.md (hunk 2 above)
Comment (nerrons) [reply README.md:38]: Sounds nice, as your suggestion aligns with the "include/ directories match project name" best practice

**Review by TheLartians — APPROVED** [review]

**Pull request merged**